webapp/crashstats/crashstats/admin.py

```diff
@@ -30,14 +30,14 @@ class LogEntryAdmin(admin.ModelAdmin):
 
     list_display = [
         "action_time",
-        "admin",
+        "admin_email",
         "object_link",
         "action",
         "get_change_message",
     ]
     list_display_links = ["action_time", "get_change_message"]
 
-    def admin(self, obj):
+    def admin_email(self, obj):
```

Comment:
We needed to change this method name because it overrides `admin` in the class namespace preventing us from using `admin.display()` decorator.